### PR TITLE
[IDB IGNORE] [MDB IGNORE] [Ready to Merge] More euthenia changes

### PR DESCRIPTION
**Pull request by ktoma36**
## About The Pull Request

More  Euthenia changes
## Why It's Good For The Game

Progress is good

### CODE REVIEW

File: code/modules/mapping/map_helpers/network_builder/atmos_pipes.dm [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	if(!network_directions)
+		return
+		/*
+	var/knot = (src.knot == KNOT_FORCED) || ((src.knot == KNOT_AUTO) && detect_knot())
+	if(knot)
+		for(var/i in GLOB.cardinal)
+			if(!(network_directions & i))
+				continue
+			new /obj/structure/cable(loc, capitalize(cable_color), 0, i, TRUE)
+			*/
```
Comment (silicons) [inline code/modules/mapping/map_helpers/network_builder/atmos_pipes.dm:45]: dont leave in comments like this

File: code/modules/mapping/map_helpers/network_builder/atmos_pipes.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			. |= i
+			continue
+		var/opp = turn(i, 180)
+		for(var/obj/machinery/atmospherics/pipe/simple/P in T)
+			if((P.initialize_directions == opp) && P.color == pipe_color)
+				. |= i
+				continue
+
+/obj/map_helper/network_builder/atmos_pipe/build()
+	if(!network_directions)
```
Comment (silicons) [inline code/modules/mapping/map_helpers/network_builder/atmos_pipes.dm:36]: uhh this is just a copypaste of cables?

Comment (ktoma36) [reply code/modules/mapping/map_helpers/network_builder/atmos_pipes.dm:36]: Yeah, I was attempting to try and make it work for pipes but my brain is too much goo. I'll delete the file when i get home :)

File: code/modules/xenobio/items/special_slime_objects.dm [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+//	icon_state = "slime cube"
+	if(searching == 1)
+		searching = 0
+		var/turf/T = get_turf_or_move(src.loc)
+		for (var/mob/M in viewers(T))
+			M.show_message("<span class='warning'>The activity in the slime core dies down. Maybe it will stir another time.</span>")
+
+/obj/item/slime_cube/transfer_personality(var/mob/candidate)
+	announce_ghost_joinleave(candidate, 0, "They are a promethean now.")
+	src.searching = 2
```
[the remark this answers is not in the document]
Comment (ktoma36) [reply code/modules/xenobio/items/special_slime_objects.dm:52]: It's mostly a copy pasta of the slime cube with different flavor text. Meant for special events and for the Euthenia lore shenanigans

[the remark this answers is not in the document]
Comment (ktoma36) [reply code/modules/xenobio/items/special_slime_objects.dm:52]: Oh and ye i can make it into its own or

**Review by silicons — APPROVED** [review]

**Pull request merged**